jekyll/_cci2/configuration-reference.md

```diff
@@ -845,8 +845,12 @@ If you are using CircleCI server, contact your system administrator for details
 
 When using the Linux [GPU executor]({{ site.baseurl }}/using-gpu), the available images are:
 
-* `linux-cuda-11:default` v11.4, v11.6, v11.8 (default), Docker v20.10.24
-* `linux-cuda-12:default` v12.0, v12.1 (default), Docker v20.10.24
+* `ubuntu-2004-cuda-11.4:202110-01` - CUDA v11.4.2, Docker v20.10.7, nvidia-container-toolkit v1.5.1-1
```

Comment:
Something odd going on here @kbatuigas - these changes were in master and now they are not. Must be something that crept back during the GH apps release I think. Do you know who worked on these changes originally so we can check the content is correct?

Reply:
Oh @JalexChen could you check? Looks like this branch somehow is trying to revert your recent changes. Can you confirm the text in the red here is actually correct and what we want?